imagedephi/main.py

```diff
@@ -24,12 +27,31 @@ def imagedephi() -> None:
     "input-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
 )
 @click.argument(
-    "output-dir", type=click.Path(exists=True, file_okay=False, writable=True, path_type=Path)
+    "output-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
 )
-def run(input_dir: Path, output_dir: Path) -> None:
-    """Run in CLI-only mode."""
-    redact_images(input_dir, output_dir)
-    click.echo("Done!")
+@click.argument("override-rules", type=click.File("r"), required=False)
+def run(input_dir: Path, output_dir: Path, override_rules: Optional[TextIO]):
+    """Redact images in a folder according to given rule sets."""
+    base_rules_bytes = pkgutil.get_data("imagedephi", "base_rules.yaml")
+    if base_rules_bytes is None:
+        raise click.ClickException("There was an error with finding the base rules.")
```

Comment:
```suggestion
```

If this happens, something is catastrophically wrong with our packaging, so it's fine to let errors naturally get thrown by whatever is actually trying to do the reading.

Reply:
Makes sense to me. This was removed as part of transitioning to `importlib.resources`. (c234aa9)